### PR TITLE
Obsolete omiabis.md

### LINKED ISSUE #1442: Obsolete OMIABIS
The namespace is now owned by OBIB.

### PR DESCRIPTION
**Pull request by matentzn**
Fixes #1442

@mbrochhausen can you double check?

### CODE REVIEW

**Comment by mbrochhausen** [comment]
I am sure I can do something, but I am not sure what to check for. What
precisely do you need me to do?

File: ontology/omiabis.md
Comment (cthoyt) [inline ontology/omiabis.md:20]: Can we make a structured field for tracking when one ontology is merged into another

Comment (cthoyt) [reply ontology/omiabis.md:20]: Also if the repo is obsolete, GitHub has a way of "archiving" the repo as well. Should we ask that people do that (if there's even a contact person who could?)

**Comment by matentzn** [comment]
@mbrochhausen just check whether these edits look reasonable to you: https://github.com/OBOFoundry/OBOFoundry.github.io/pull/1547/files

File: ontology/omiabis.md
Comment (matentzn) [reply ontology/omiabis.md:20]: I think asking people to do that is a good idea! @mbrochhausen would you consider trying to archive the OMIABiS repo?

**Comment by mbrochhausen** [comment]
Yes, those look fine.

File: ontology/omiabis.md
Comment (cthoyt) [reply ontology/omiabis.md:20]: Instructions on how to archive a repo: https://github.blog/2017-11-08-archiving-repositories/

**Comment by matentzn** [comment]
I need an approval to merge. Thank you! @cthoyt

**Comment by matentzn** [comment]
Oh, you can do it as well, thank you! Click here:
https://github.com/OBOFoundry/OBOFoundry.github.io/pull/1547/files

Then Review Changes, then "Approve" then ok.

**Review by mbrochhausen — APPROVED** [review]

**Comment by mbrochhausen** [comment]
done.

**Review by cthoyt — APPROVED** [review]
LGTM

however, is it necessary to delete the tracker? Just because the ontology is marked as obsolete, it doesn't necessarily mean people wouldn't have reasons to get in touch with the old maintainers. Worst case scenario, the issue tracker can at least list known issues or discussions even if they will never be addressed

**Comment by matentzn** [comment]
Ok good point @cthoyt  you need to approve again.

**Review by cthoyt — APPROVED** [review]

**Comment by matentzn** [comment]
Someone else needs to merge this, seems like @cthoyt  and @mbrochhausen votes don't count somehow.

**Review by balhoff — APPROVED** [review]

**Comment by cthoyt** [comment]
@matentzn such are the pains of an expat in europe :p

github said that it needed a review from someone with commit privileges

**Pull request merged**
